### DISCUSSION TITLE
how to use pluginmessages packets in minestom

### DISCUSSION THREAD

**Discussion by bebranator** [Minestom/Minestom#1915] [Help]
how to send it from client and make a server understand it ( using specified channel )

i'm dummy idk how to use pluginmessages

**Comment by Tofaa2** [comment]
Player#sendPluginMessage

**Reply by bebranator** [reply]
how to make client send it plz
forgg

**Comment by UserUNP** [comment]
https://wiki.vg/Plugin_channels
